.github/workflows/ubuntu_24.04/build.sh

```diff
@@ -5,10 +5,13 @@ set -e
 LD_LIBRARY_PATH="/opt/instantclient_19_9:/opt/instantclient_19_9/lib:${LD_LIBRARY_PATH}"
 export LD_LIBRARY_PATH
 
+GDAL_PYTHON_CXXFLAGS="-DBANDMAP_TYPE_CONST_SAFE"
```

Comment:
Perhaps just rename it to `GDAL_BANDMAP_CONST_SAFE` to make it immediately clear it's a GDAL flag/macro

Reply:
done